### ISSUE TITLE
huh? middleware?

### ISSUE THREAD

**Issue by caarlos0** [charmbracelet/wish#236]
it should be possible to run it wrapped into another model, but it would be nice to have a simple way to "just serve this form for me".

**Comment by caarlos0** [comment]
pending on some decisions on how to pass down the lipgloss renderer

**Comment by caarlos0** [comment]
I think this can be closed: https://github.com/charmbracelet/huh/pull/216
